lua/cmp_fuzzy_path/init.lua

```diff
@@ -92,7 +90,7 @@ source.complete = function(self, params, callback)
       callback({ items = {}, isIncomplete = true })
       return
     end
-    if COMMAND_SHORTCUT:match_str(params.context.cursor_before_line) then
+    if vim.regex(opts.cmd_trigger_regex):match_str(params.context.cursor_before_line) then
```

Comment:
again, need to cache these regexs locally.

Reply:
Ok I guess we can use a table with the original regex string as keys and vim.regex lua data as the values, but why use vim.regex here? Is there any advantage over vim.fn.match?